app/collins/models/conversions.scala

```diff
@@ -136,7 +137,10 @@ object conversions {
       (json \ "UPDATED").asOpt[Timestamp],
       (json \ "DELETED").asOpt[Timestamp],
       (json \ "ID").as[Long],
-      (json \ "STATE").asOpt[State].map(_.id).getOrElse(0)))
+      (json \ "STATE") match {
+        case JsNull => State.ANY_STATUS
```

Comment:
I don't understand how state can be null since it should be defined once the asset is created unless Unknown translates to null???